### DISCUSSION TITLE
Export data to external API

### DISCUSSION THREAD

**Discussion by oachoor** [ActivityWatch/activitywatch#867] [Q&A]
Hello,

Is there any way to transport the activity data to an external API, sort of web-hook or a worker on hourly/daily basis

Thanks.

**Comment by BelKed** [comment]
Yup, ActivityWatch has an [API](https://docs.activitywatch.net/en/stable/api.html) that you can use to export data from the database.
There is also a [Python package](https://pypi.org/project/aw-client) which makes the work with the API easier.

So now, programming a script that would periodically send data to an external API shouldn’t be that hard…
If you have any questions, feel free to ask :)

**Comment by airtonix** [comment]
We should definitely make this a primary documented feature.

Most companies that want you to track your time, want you to use a service that doesn't provide an adequate experience.

Being able to have some recommended way to export stuff would be great. 

I'd even go as far as to say that activity-watch should do the schedule exporting for us if we were to configure a plugin that does this.
